src/main/java/com/mraof/minestuck/inventory/musicplayer/CapabilityProviderMusicPlayer.java

```diff
@@ -0,0 +1,58 @@
+package com.mraof.minestuck.inventory.musicplayer;
+
+import net.minecraft.core.Direction;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.ICapabilityProvider;
+import net.minecraftforge.common.util.INBTSerializable;
+import net.minecraftforge.common.util.LazyOptional;
+import net.minecraftforge.items.CapabilityItemHandler;
+import net.minecraftforge.items.IItemHandler;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class CapabilityProviderMusicPlayer implements ICapabilityProvider, INBTSerializable<CompoundTag>
```

Comment:
Shouldn't `MusicPlayer` be a prefix here?

Reply:
Yes, im changing it